src/shared/install.h

```diff
@@ -77,8 +77,12 @@ enum UnitFileChangeType {
         _UNIT_FILE_CHANGE_TYPE_INVALID = -1
 };
```

Comment:
I figure the _UNIT_FILE_CHANGE_TYPE_INVALID definition should go away, because it is equivalent with -EPERM now...